chainstate/tx-verifier/Cargo.toml

```diff
@@ -8,18 +8,19 @@ rust-version.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-chainstate-storage = {path = '../storage'}
-chainstate-types = {path = '../types'}
-common = {path = '../../common'}
-consensus = {path = '../../consensus'}
-constraints-value-accumulator = {path = '../constraints-value-accumulator'}
-crypto = {path = '../../crypto'}
+chainstate-storage = {path = "../storage"}
+chainstate-types = {path = "../types"}
+common = {path = "../../common"}
+consensus = {path = "../../consensus"}
+constraints-value-accumulator = {path = "../constraints-value-accumulator"}
+crypto = {path = "../../crypto"}
 logging = { path = "../../logging" }
-pos-accounting = {path = '../../pos-accounting'}
+pos-accounting = {path = "../../pos-accounting"}
+randomness = { path = "../../randomness" }
 serialization = { path = "../../serialization" }
-tokens-accounting = {path = '../../tokens-accounting'}
-utils = {path = '../../utils'}
-utxo = {path = '../../utxo'}
+tokens-accounting = {path = "../../tokens-accounting"}
+utils = {path = "../../utils"}
+utxo = {path = "../../utxo"}
```

Comment:
While we're at reformatting `Cargo.toml`, most entries have spaces on the inner side of the curly braces.

Reply:
There's quite a few, here's a handy command:
```
rg -g 'Cargo.toml' '\{path|"}'
```